packages/realm-server/tests/indexing-test.ts

```diff
@@ -835,6 +839,15 @@ module('indexing', function (hooks) {
       assertCssDependency(entry.deps, pattern, fileName);
     });
   });
+
+  test('will not invalidate non-json/non-executable files', async function (assert) {
+    let deletedEntries =
```

Comment:
@habdelra I am not sure if this is a good way to test this, is it possible to test the invalidations discovery more directly?

Reply:
you should be able to just connect to the DB and see if there are files that are neither JSON nor executable in the boxel_index table. the `dbAdapter` should be inscope. just execute some SQL to see what is in the DB